src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+  const errorMessage = 'Bzzz... Error!';
+  const getSum = (x) => {
+    return function(y) {
+      callCount++;
+
+      if (callCount < 4 || (callCount % 2 !== 0)) {
```

Comment:
```suggestion
      if (callCount <= 3 || callCount % 2 !== 0) {
```
Використати цифру 3 було б більш логічно, якщо відштовхуватись від умови задачі